### PR TITLE
Return statuses on duties

### LINKED ISSUE #5002: validator node not providing feedback when a secondary key's status changes
i might be completely off base on this one, but the following seems to happen:
- assuming one key active 0x0000 and one just created but deposit not sent yet 0x1111
- WaitForActivation runs and returns as soon as it can do something which results in logs for 0x0000 as active and 0x1111 as UNKNOWN_STATUS
- send deposit for 0x1111, no feedback in logs
- wait some more, still no mention of any change of status
logs would look like 
```
[2020-03-04 19:23:14]  INFO validator: New assignment attesterSlot=394018 committeeIndex=0 epoch=12313 pubKey=0x000 status=ACTIVE validatorIndex=44819
[2020-03-04 19:23:14]  INFO validator: New assignment committeeIndex=0 epoch=12313 pubKey=0x1111 status=UNKNOWN_STATUS validatorIndex=0
```
which would also be misleading as 0x1111 might be DEPOSITED or PENDING

thanks guys for all the hard work and sorry if i missed something obvious

### PR DESCRIPTION
**Pull request by tzapu**
Resolves #5002 
---

# Description
- adds metrics for validator key statuses
- returns correct status on GetDuties for DEPOSITED as well
- updated pkey to pubkey on all metrics (suggested by @rauljordan)

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Tests failing. Please fix.

**Comment by tzapu** [comment]
i think i sorted it... lemme know if there's anything else
impressive test suite btw 👑 

this can't be right tho... can it?
image

File: validator/client/validator.go [hunk 1]
```diff
@@ -49,6 +51,18 @@ type validator struct {
...
 }
 
+var validatorStatusesGaugeVec = promauto.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "validator",
+		Name:      "statuses",
+		Help:      "validator statuses: 0 UNKNOWN, 1 DEPOSITED, 2 PENDING, 3 ACTIVE, 4 EXITING, 5 SLASHING, 6 EXITED",
+	},
+	[]string{
+		// validator pubkey
```
Comment (terencechain) [inline validator/client/validator.go]:
```suggestion
		// Validator pubkey.
```

**Comment by rauljordan** [comment]
Thanks @tzapu ! Yeah codecov is pretty inaccurate so good to ignore for now

**Pull request merged**